client/my-sites/site-settings/cloudflare.js

```diff
@@ -18,8 +20,12 @@ const Cloudflare = () => {
 	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) ) || 0;
 	const sitePlan = useSelector( ( state ) => getSitePlanSlug( state, siteId ) );
```

Comment:
Since `sitePlan` is not used anymore we should be able to remove all associated code (including `QuerySitePlans`).
Worth a consideration if we should replace `QuerySitePlans` with `QuerySiteFeatures`

Reply:
Great catch! I've updated it.